GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -24,48 +24,57 @@ function UnloadKext()
 
 function UnInstallVFSForGit()
 {
-    if [ -d "${VFSFORDIRECTORY}" ]; then
-        rmCmd="sudo /bin/rm -Rf ${VFSFORDIRECTORY}"
-        echo "$rmCmd..."
-        eval $rmCmd || exit 1
-    fi
-    
     if [ -d "${PRJFSKEXTDIRECTORY}/$KEXTFILENAME" ]; then
         rmCmd="sudo /bin/rm -Rf ${PRJFSKEXTDIRECTORY}/$KEXTFILENAME"
         echo "$rmCmd..."
-        eval $rmCmd || exit 1
+        eval $rmCmd || { echo "Error: Could not delete ${PRJFSKEXTDIRECTORY}/$KEXTFILENAME. Delete it manually."; exit 1; }
     fi
     
     if [ -f "${LAUNCHDAEMONDIRECTORY}/$LOGDAEMONLAUNCHDFILENAME" ]; then
-        unloadCmd="sudo launchctl unload -w ${LAUNCHDAEMONDIRECTORY}/$LOGDAEMONLAUNCHDFILENAME"
-        echo "$unloadCmd..."
-        eval $unloadCmd || exit 1
+        isLoadedCmd="sudo launchctl kill SIGCONT system/org.vfsforgit.prjfs.PrjFSKextLogDaemon"
+        echo "$isLoadedCmd"
+        if $isLoadedCmd; then
+            unloadCmd="sudo launchctl unload -w ${LAUNCHDAEMONDIRECTORY}/$LOGDAEMONLAUNCHDFILENAME"
+            echo "$unloadCmd..."
+            eval $unloadCmd || { echo "Error: Could not unload ${LAUNCHDAEMONDIRECTORY}/$LOGDAEMONLAUNCHDFILENAME. Unload it manually (\"$unloadCmd\")."; exit 1; }
+        fi
+        
         rmCmd="sudo /bin/rm -Rf ${LAUNCHDAEMONDIRECTORY}/$LOGDAEMONLAUNCHDFILENAME"
         echo "$rmCmd..."
-        eval $rmCmd || exit 1
+        eval $rmCmd || { echo "Error: Could not delete ${LAUNCHDAEMONDIRECTORY}/$LOGDAEMONLAUNCHDFILENAME. Delete it manually."; exit 1; }
     fi
-    
+        
     # Unloading Service LaunchAgent for each user
     # There will be one loginwindow instance for each logged in user, 
     # get its uid (this will correspond to the logged in user's id.) 
     # Then use launchctl bootstrap gui/uid to auto load the Service 
```

Comment:
Update comment to describe what we expect the isLoadedCmd to do? Also, this is old but we should fix the copy/paste error from the preinstall script:
> Then use launchctl bootstrap gui/uid to auto load the Service 

That should be bootout.

Reply:
rephrased